desktop/apps/auction/components/layout/Banner.js

```diff
@@ -51,6 +52,16 @@ function Banner (props) {
               </a>
             </div>
           )
+        } else if (isClosed) {
+          return (
+            <div className={b('closed')}>
+              <div>
+                Auction Closed
+              </div>
+            </div>
+          )
+        } else if (isEcommerceSale) {
+          return null
```

Comment:
E-commerce sales will still generally have an `end_at` which we will want to show in the banner (and if it is closed, we'll probably just want to say "Sale Closed")